### PR TITLE
Fix: Cmds and project cleanup

### PR DESCRIPTION
**Pull request by Alf21**
- Fixed `kill` cmd
- Added `Respawn` and `PlayerID` command

### CODE REVIEW

File: code/commands/PlayerCommands.cs [hunk 1]
```diff
@@ -0,0 +1,44 @@
+using System.Collections.Generic;
+using System.Linq;
+using Sandbox;
+
+using TTTReborn.Player;
+
+namespace TTTReborn.Commands
+{
+    public static class PlayerCommands
```
Comment (TimGoll) [inline code/commands/PlayerCommands.cs:9]: indentation of namespace wrong, also remember the empty line in front of the closing namespace bracket.

File: code/commands/PlayerCommands.cs [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+    {
+        [ServerCmd("Respawn")]
+        public static void RespawnPlayer(int id)
+        {
+            List<Client> playerList = Client.All.ToList();
+
+            for (int i = 0; i < playerList.ToList().Count; i++)
+            {
+                if (i == id)
```
Comment (TimGoll) [inline code/commands/PlayerCommands.cs:18]: maybe use fencing here?

File: code/commands/PlayerCommands.cs [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+
+        [ClientCmd("PlayerID")]
+        public static void PlayerID()
+        {
+            List<Client> playerList = Client.All.ToList();
+
+            for (int i = 0; i < playerList.ToList().Count; i++)
+            {
+                if (playerList[i] == null)
+                    continue;
```
Comment (TimGoll) [inline code/commands/PlayerCommands.cs:38]: no conditional statements without brackets

**Review by TimGoll — CHANGES_REQUESTED** [review]
Besides the mentioned things, shouldn't commands be all lowercase? So `respawn` instead of `Respawn`?

File: .editorconfig [hunk 4]
```diff
@@ -0,0 +1,12 @@
...
+# top-most EditorConfig file
+root = true
+
+[*]
+indent_style = space
+indent_size = 4
+end_of_line = crlf
+charset = utf-8
+trim_trailing_whitespace = false
+insert_final_newline = false
```
Comment (Histalek) [inline .editorconfig:12]:
```suggestion
trim_trailing_whitespace = true
insert_final_newline = true

[*.md]
insert_final_newline = false
trim_trailing_whitespace = false
```

maybe this is what you wanted to achieve? i would definitely prefer it this way

File: code/commands/PlayerCommands.cs [hunk 5]
```diff
@@ -0,0 +1,42 @@
+using System.Collections.Generic;
+using System.Linq;
+using Sandbox;
+
+using TTTReborn.Player;
+
+namespace TTTReborn.Commands
+{
+public static class PlayerCommands
+{
```
Comment (Histalek) [inline code/commands/PlayerCommands.cs:10]: i'm still not sure how i feel about the non-indentation for the namespace :/ i get the 'unneeded indentation' point but it still looks exceedingly weird

**Review by Histalek — CHANGES_REQUESTED** [review]
some nitpicks about the editorconfig and indentation in general, up for discussion.

Also indentation seems to be borked right now, i see a lot of 5 space indentation and also some tabs. you might want to look into that

File: .editorconfig [hunk 6]
```diff
@@ -0,0 +1,12 @@
+# EditorConfig is awesome: https://EditorConfig.org
+
+# top-most EditorConfig file
+root = true
+
+[*]
+indent_style = space
+indent_size = 4
+end_of_line = crlf
```
[the remark this answers is not in the document]
Comment (Histalek) [reply .editorconfig]: resolved by 25b9b22

**Review by Histalek — APPROVED** [review]

**Review by TimGoll — APPROVED** [review]

**Pull request merged**